asynq/scheduler.py

```diff
@@ -574,6 +575,9 @@ def extract_futures(value, result):
     elif type(value) is tuple or type(value) is list:
         for item in value:
             extract_futures(item, result)
+    elif type(value) is dict:
+        for value in six.iteritems(value):
```

Comment:
Good points, I tested using python3 and were not aware of the python2 differences or how to use `six`